src/commands/guildLog.ts

```diff
@@ -0,0 +1,189 @@
+import { ApplicationCommandOptionType, EmbedBuilder } from "discord.js";
+import { Command } from "../interfaces/Command";
+import { BotEvents } from "mineflayer";
+
+export default {
+	data: {
+		name: "guildlog",
+		description: "Get the guild log!",
+		options: [
+			{
+				name: "member",
+				description: "What user do you want to check? - Optional",
+				type: ApplicationCommandOptionType.String,
+			},
+			{
+				name: "page",
+				description: "What page do you want to check? - Optional",
+				type: ApplicationCommandOptionType.String,
+			},
+		],
+	},
+	run: async (bot, interaction, args) => {
+		const member: string = args[0];
```

Comment:
For clarity, prefer the `as` keyword for casts

Reply:
Explain?

Reply:
Typescript contains an `as` keyword to 'cast' types (or at least the language's type coverage) from one type to another. Here `args` is of type `any[]`, meaning its items could be any type, but really we know one is a string, and one is a number.

So, to make it more clear that you're actually casting the variables, and not just explicitly typing them for no reason, you could use something like the following:
`const member = args[0] as string;`